src/transformers/models/albert/configuration_albert.py

```diff
@@ -103,51 +109,51 @@ class AlbertConfig(PretrainedConfig):
     >>> configuration = model.config
     ```"""
 
+    vocab_size: int = validated_field(interval(min=1), default=30000)
+    embedding_size: int = validated_field(interval(min=1), default=128)
+    hidden_size: int = validated_field(interval(min=1), default=4096)
+    num_hidden_layers: int = validated_field(interval(min=1), default=12)
+    num_hidden_groups: int = validated_field(interval(min=1), default=1)
+    num_attention_heads: int = validated_field(interval(min=0), default=64)
+    intermediate_size: int = validated_field(interval(min=1), default=16384)
+    inner_group_num: int = validated_field(interval(min=0), default=1)
+    hidden_act: str = validated_field(activation_fn_key, default="gelu_new")
```

Comment:
if `activation_fn_key` is a key, I would rather use `ActivationFnKey` as a class 
```suggestion
    hidden_act: str = check_activation_function(default="gelu_new")
```

Reply:
In python 3.11, we can define something like `ActivationFnKey = Literal[possible values]`, and then simply have

```py
hidden_act: ActivationFnKey = "gelu_new"
```

Until then, I think it's best to keep the same pattern as in other validated fields, i.e. `validated_field(validation_fn, default)`